src/backend/interfaces/REST/go.server/middlewares/authentication.go

```diff
@@ -25,9 +28,14 @@ func BasicAuthFromCache(cache *redis.Client, realm string) gin.HandlerFunc {
 		// Get provided auth headers
 		var user_id, access_token string
 		var ok bool
-		if user_id, access_token, ok = c.Request.BasicAuth(); !ok {
-			kickUnauthorizedRequest(c, realm)
+
+		//Try auth-scheme 'Bearer' then 'Basic'
+		if user_id, access_token, ok = BearerAuth(c.Request); !ok {
+			if user_id, access_token, ok = c.Request.BasicAuth(); !ok {
```

Comment:
The question is: should we allow multiple ways to authenticate on the backend ?
So this must be documented.

IMO, Bearer does the job and should be the only way (api/v1& api/v2).
If Basic is just a convenient way to authenticate for tests purpose, this should not be available on production code: We may provide an other tool to generate a Bearer (the actual express middleware may be)